src/components/MdTable/MdTable.vue

```diff
@@ -332,7 +335,8 @@
         this.$emit('update:mdSelectedValue', val)
         this.$emit('md-selected', val)
       },
-      syncSelectedValue () {
+      syncSelectedValue: async function () {
+        await this.$nextTick() // render the table first
```

Comment:
As @Samuell1 said https://github.com/vuematerial/vue-material/pull/1874#issuecomment-407495079
would you change it to `this.$nextTick(() => { /* ... code here */  })`?

Additionally, I think this `$nextTick` could be removed with this fix.
https://github.com/rayfoss/vue-mater/blob/42ce8fcc2ebf38986d9d09d5cf296ecce93212d3/src/components/MdTable/MdTable.vue#L357

Reply:
You're right about the created() -> nextTick(), it became superfluous and the bug fix still works on all cases without it.